source/_docs/guides/bitbucket.md

```diff
@@ -0,0 +1,199 @@
+git remote add pantheon $PANTHEON_REPO
+git push --force pantheon master
+```
+
+In the terminal, you should see Pantheon applying the pantheon.yml file configuration. When this is complete, the site should have a single commit, and be in sync with the Bitbucket repo. You should be able to view the site:
+
+```
+terminus env:view $SITE.dev
+```
+
+## Step 4: Create CircleCI Authentication
+
+Next, we will connect CircleCI to access both Bitbucket and Pantheon. For security purposes, we will create and configure a Pantheon user solely for CircleCI continuous integration purposes.  
+
+1. Using a unique email address, create a new Pantheon user from the [Pantheon home page](https://pantheon.io/).
+
+2. [Create an SSH key](https://pantheon.io/docs/ssh-keys) for this user.
+
+3. [Add this SSH public key](https://pantheon.io/docs/ssh-keys/#add-your-ssh-key-to-pantheon) to the user's dashboard.
+
+![Add the public key to user dashboard](/source/docs/assets/images/integrations/bitbucket/ssh_key.png)
+
+4. Add this user to your site's team.
+
+5. On this user's account page, go to the Machine Token tab on the left navigation, and create a machine token, which is a string of text, for CircleCI to authenticate with. Save this token somewhere temporarily.
+
+![Create a machine token from user dashboard](/source/docs/assets/images/integrations/bitbucket/machine_token.png)
+
+
+## Step 5: Configure CircleCI
+
+1. Log into CircleCi and click on Projects on the left navigation. Authorize Bitbucket access.
+
+![Authorize Bitbucke repo access](/source/docs/assets/images/integrations/bitbucket/circle_bb.png)
+
+Create a new project based on the Bitbucket repo you created. This project will run and fail.
+
+![CFirst build](/source/docs/assets/images/integrations/bitbucket/circle_first_build.png)
+
+3. Add the CircleCI user's private key to the SSH Permissions tab in the project.
+
+![Add private key](/source/docs/assets/images/integrations/bitbucket/private_key.png)
+
+4. Create the following environmental variables within the project:
+
+- TERMINUS_SITE: The name of the Pantheon site.
+- TERMINUS_TOKEN: The Terminus Machine token that you created.
+- TEST_SITE_NAME: Used to set the name of the test site when installing Drupal.
+- ADMIN_EMAIL: Used to configure the email address to use when installing Drupal.
+- ADMIN_PASSWORD: Used to set the password for the UID 1 user during site installation.
+- GIT_EMAIL: Used to configure the git user’s email address for commits we make.
+
+Now you can rerun the failed job again from the Builds tab in CircleCI and you can follow the build from the dashboard. This time it should pass. 
+
+![Running tests](/source/docs/assets/images/integrations/bitbucket/running.png)
+
+## Final Step: Test the New Workflow
+Now try to submit a pull request. Let's add something to the repo and test the workflow. From the terminal:
+
+```
+git checkout -b test
+vim tests/features/content-ui.feature
```

Comment:
I'll handle style issues like that during the copy edit, don't worry about it for now. So long as the tech is correct, I'll make it pretty later 😉.